cmd/tikv-ctl/src/cmd.rs

```diff
@@ -600,6 +600,33 @@ pub enum Cmd {
         #[structopt(long, default_value = "symlink")]
         rocksdb_files: String,
     },
+    /// flashback data in cluster to a certain version
+    ///
+    /// NOTE: Should use `./pd-ctl config set halt-scheduling true` to halt PD
```

Comment:
As we already pass the pd address, can we trigger the halt in tikv-ctl flashback code?

Reply:
Good catch! But I think it's better to do it at the next pr.